packages/std/src/lib.rs

```diff
@@ -1,15 +1,40 @@
-// these two are conditionally compiled, only for wasm32
-pub mod exports;
-pub mod imports;
-
-pub mod encoding;
-pub mod errors;
-pub mod memory;
-pub mod mock;
-pub mod serde;
-pub mod storage;
-pub mod traits;
-pub mod types;
-
-// not exposed
+// Exposed on all platforms
+
+mod encoding;
+mod errors;
+mod mock;
+mod serde;
+mod storage;
+mod traits;
+mod types;
+
+pub use crate::encoding::Binary;
+pub use crate::errors::{
+    contract_err, dyn_contract_err, invalid, unauthorized, Error, NotFound, NullPointer, ParseErr,
+    Result, SerializeErr,
+};
+pub use crate::mock::{mock_dependencies, mock_env, MockApi, MockStorage};
```

Comment:
As mentioned above, I like all testing-related traits somehow in a labeled package `mock`, `testing`.
I guess the name prefixes make sense. I also like namespacing warnings, it makes it more clear this is not to be used in production code. I'd like to hear your reasoning why none?

Reply:
What do you mean by "namespacing warnings"?

Reply:
```rust
use cosmwasm::{coin, contract_err, mock_env};
```

seems easier to overlook than

```rust
use cosmwasm::{coin, contract_err,};
use cosmwasm::testing::mock_env;
```

Forcing them to be on different lines and much more explicit

Reply:
Agreed. Done in https://github.com/CosmWasm/cosmwasm/pull/177/commits/6656ccadc985aac4f72058428e14d174061fcb58